sirepo/job_api.py

```diff
@@ -69,6 +71,14 @@ def api_downloadDataFile(simulation_type, simulation_id, model, frame, suffix=No
         )
 
 
+@api_perm.require_user
+def api_jobs():
+    req = sirepo.http_request.parse_post().req_data
+    if req.adm:
```

Comment:
This is an unchecked value, and it is related to security so that's potentially dangerous. The value of `api_admJobs` was parsed by `uri_router` and validated against the schema in order to find the function. That's an important check. Next, if we decide to implement api-based roles, we would want api_admJobs to be distinct from api_ownJobs so that the role checks would be distinct.

Also,`req` is always the result of `parse_post`, not `req_data`. To refer to `req_data` directly, use `req.req_data`. Naming consistency is important.